Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-AlertsAndConfirmationDialogs.swift

```diff
@@ -89,14 +89,18 @@ struct AlertAndConfirmationDialogView: View {
   var body: some View {
     WithViewStore(self.store) { viewStore in
       Form {
-        Section(header: Text(template: readMe, .caption)) {
-          Text("Count: \(viewStore.count)")
-          Button("Alert") { viewStore.send(.alertButtonTapped) }
-          Button("Confirmation Dialog") { viewStore.send(.confirmationDialogButtonTapped) }
+        Section {
+          DisclosureGroup("About this case study") {
+            Text(template: readMe)
+          }
```

Comment:
very tiny suggestion, but should we just have a `about(String) -> some View` function that does this, or an `AboutView`? just to considerate the "About this case study" and whatnot?